src/NDItk/multigroup/ReactionCrossSections/src/verify.hpp

```diff
@@ -0,0 +1,31 @@
+/**
+ *  @brief Verify the groups structure
```

Comment:
These comments are for the group boundaries function.

Reply:
Gotta love copy/paste. Fixed in fix/review.